memory/database.go

```diff
@@ -227,14 +228,41 @@ func (d *BaseDatabase) AddTable(name string, t MemTable) {
 	d.tables[name] = t
 }
 
+func BacktickDefaultColumnValueNames(sch sql.Schema) sql.Schema {
```

Comment:
is there a dependency cycle if this is in planbuilder?

Reply:
yes